core/translations/i18n/en.json

```diff
@@ -49,82 +49,7 @@
   "Unloading module \"${0}\".": "Unloading module \"${0}\".",
   "What kind of shit platform is this that doesn't even support sending messages?": "What kind of shit platform is this that doesn't even support sending messages?",
   "Working on it...": "Working on it...",
-
```

Comment:
48? Yeah let's keep it this way then 😆 ...as a memento to the original name as well